native/components/Navbar.jsx

```diff
@@ -0,0 +1,77 @@
+//utilities
+import * as React from 'react';
```

Comment:
this is dangerous! we're importing everything in the React library instead of: `import {specificComponent} from 'React' meaning that this file alone is can be super beefy. 
I also noticed that we don't use `React.someMethod` anywhere in this file, so this line is actually stale code